tools/sgen/input/FluentUIStyle.yml

```diff
@@ -320,6 +320,13 @@ AP_MSFCommandBarTokens:
   itemIconColor: [ rest: $Colors.Foreground.neutral1, hover: $Colors.Foreground.neutral1, pressed: $Colors.Foreground.neutral1, selected: $Colors.Foreground.neutralInverted, disabled: $Colors.Foreground.neutralDisabled ]
   itemInterspace: $Spacing.xxxSmall
 
+AP_MSFDividerTokens:
+  color: [ rest: $Colors.Stroke.neutral1]
```

Comment:
This was the color from the Figma spec (List -> Border), but it's even harder to see in the test app than it was in the spec. We should probably review the spec with design

Reply:
That's a great point! Thanks for bringing it up.
Do you think that could bring the number of divider styles to just one? It'd be great to save on definitions of tokens for it.

Reply:
Last time I spoke with the previous designer, they emphasized that they wanted to replicate how the UIKit separator looks. We could try doing a color comparison and seeing if they are open to darkening the Divider color so that it better matches the legacy color?